nucleus/pom.xml

```
@@ -119,7 +119,7 @@
 
         <!-- Servlet -->
         <!-- Requires JDK8u161 or above-->
-        <grizzly.npn.api.version>2.0.0</grizzly.npn.api.version>
+        <grizzly.npn.api.version>2.0.1</grizzly.npn.api.version>
```

We're using a patched version.
```suggestion
        <grizzly.npn.api.version>2.0.0.payara-p1</grizzly.npn.api.version>
```